src/main/kotlin/it/pagopa/ecommerce/scheduler/client/PaymentGatewayClient.kt

```diff
@@ -21,14 +21,20 @@ class PaymentGatewayClient {
 
 
     fun requestRefund(requestId: UUID): Mono<PostePayRefundResponseDto> {
-        return paymentTransactionsControllerApi.refundRequest(requestId)
+/*        return paymentTransactionsControllerApi.refundRequest(requestId)
             .onErrorMap(WebClientResponseException::class.java) { exception: WebClientResponseException ->
                 when (exception.statusCode) {
                     HttpStatus.NOT_FOUND -> TransactionNotFound(requestId)
                     HttpStatus.GATEWAY_TIMEOUT -> GatewayTimeoutException()
                     HttpStatus.INTERNAL_SERVER_ERROR -> BadGatewayException("")
                     else -> exception
                 }
-            }
+            }*/
+        return Mono.just(PostePayRefundResponseDto().apply {
+            this.requestId = requestId.toString()
+            paymentId = UUID.randomUUID().toString()
+            error = ""
+            refundOutcome = "OK"
+        })
```

Comment:
watch out, remove this mock :)

Reply:
Whoops! 😅

Reply:
Fixed in https://github.com/pagopa/pagopa-ecommerce-scheduler-service/pull/31/commits/afa062b65814348579d74b9ee6e4848acdd45b75